src/components/users/request-email-change-form.tsx

```diff
@@ -25,23 +30,20 @@ async function requestEmailChange(newEmail: string) {
 const RequestEmailChangeForm: React.FunctionComponent<RequestEmailChangeFormProps> = ({
   originalEmail,
 }) => {
-  const VIEW_MODE = 'VIEW_MODE'
-  const EDIT_MODE = 'EDIT_MODE'
-  const [mode, setMode] = React.useState(VIEW_MODE)
-
-  const [
-    changeRequestSubmittedFor,
-    setChangeRequestSubmittedFor,
-  ] = React.useState<string | null>(null)
+  const [state, send] = useMachine(requestEmailChangeMachine, {
+    services: {
+      requestChange: (_, event: DoneEventObject) => {
+        return requestEmailChange(event.data.newEmail)
+      },
```

Comment:
I'm not sure if I got the typing right here. The compiler didn't like `Event` type created in the other file, so I re-exported this `DoneEventObject` to satisfy it.